extras/parse_travis_log.py

```diff
@@ -177,6 +180,35 @@ def process_static_analysis(f, line):
         if line.startswith(' - clang-format for '):
             d.update(process_clang_format(f, filename))
 
+def process_pep8(f, line):
+    """
+    Process PEP8 output for a certain file.
+    """
+    # Check PEP8 on file pynest/nest/lib/hl_api_connections.py:
+    filename = line.split(' ')[-1].strip()[0:-1]
+    d = set()
+    res = {filename: d}
+    while True:
```

Comment:
Options here: 
````
fiter = iter(f)
for line in fiter:
```
and use the same iter at 206
or ... comment what's going on here with walking through the file.